lua/nvim_lsp/dartls.lua

```diff
@@ -0,0 +1,42 @@
+local nvim_lsp = require 'nvim_lsp'
+local configs = require 'nvim_lsp/configs'
+local lsp = vim.lsp
+
+local server_name = "dartls"
+local bin_name = "dart"
+local dart_sdk_path = os.getenv("DART_SDK")
+
+configs[server_name] = {
+  default_config = {
+    cmd = {bin_name, dart_sdk_path .. "/bin/snapshots/analysis_server.dart.snapshot", "--lsp"};
+    filetypes = {"dart"};
+    root_dir = nvim_lsp.util.root_pattern("pubspec.yaml");
+    log_level = lsp.protocol.MessageType.Warning;
+    init_options = {
+      onlyAnalyzeProjectsWithOpenFiles = "false",
+      suggestFromUnimportedLibraries = "true",
+      closingLabels = "true",
+      outline = "true",
+      fluttreOutline= "false"
+    };
+    settings = {};
+  };
+  docs = {
```

Comment:
If we add vscode extension's package.json path to the docs, README.md is updated automatically to show detailed configuration instructions. by GitHub action.
```suggestion
  docs = {
  	package_json = "https://raw.githubusercontent.com/Dart-Code/Dart-Code/master/package.json";
```